src/styles/blocks/products.scss

```diff
@@ -0,0 +1,9 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  width: (4 * $cardWidth) + (3 * $productDistance);
+  margin: 0 auto;
+  padding: 50px 0;
```

Comment:
<img width="915" alt="missing horizontal paddings" src="https://user-images.githubusercontent.com/39308717/201122146-89be8507-914c-46b0-b308-5e48c617c63c.png">

i guess this would solve the remaining 2 tests:
```suggestion
  padding: 50px 40px;
```